vnext/ReactUWP/Modules/Animated/FacadeType.h

```diff
@@ -19,7 +19,7 @@ enum FacadeType {
   Progress
 };
 
-static FacadeType StringToFacadeType(const std::string &string) {
+inline FacadeType StringToFacadeType(const std::string &string) {
```

Comment:
>string [](start = 56, length = 6)

it's a bit confusing to use variables same as language primitives. And although C++ "string" is not a true primitive, it's still confusing.

Reply:
The main issue here was the 'static' keyword that forces compiler create a unique code of the function. You have a good point - I will change it.

---
In reply to: [370870951](https://github.com/microsoft/react-native-windows/pull/3958#discussion_r370870951) [](ancestors = 370870951)